src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -337,7 +337,7 @@ def _generate_line_result_for_exception(self, inputs, run_id, line_number, flow_
         return result
 
     def run(self, batch_inputs):
-        signal.signal(signal.SIGINT, signal_handler)
+        # signal.signal(signal.SIGINT, signal_handler)
```

Comment:
Remove this line if it is useless instead of adding comment

Reply:
Updated.